app/src/main/java/me/hufman/androidautoidrive/carapp/music/MusicApp.kt

```diff
@@ -339,6 +343,10 @@ class MusicApp(val securityAccess: SecurityAccess, val carAppAssets: CarAppResou
 		if (enqueuedViewVisible) {
 			enqueuedView.redraw()
 		}
+		if (browseViewVisible) {
+			browseView.currentPage?.redraw()
```

Comment:
I think this should be changed to call a `browseView.redraw()` function, allowing the browseView's currentPage to be private and reducing coupling slightly.

Reply:
I agree, added a redraw method to BrowseView and calling that here